src/Lexer.php

```diff
@@ -31,6 +41,7 @@ class Lexer
      * @see https://github.com/cucumber/gherkin/blob/679a87e21263699c15ea635159c6cda60f64af3b/php/src/StringGherkinLine.php#L14
      */
     private const CELL_PATTERN = '/(?<!\\\\)(?:\\\\{2})*\K\\|/u';
+
```

Comment:
To avoid conflict with #350
```suggestion
```

Reply:
This thing about conflicts is getting a bit out of hand. I've had to solve a truckload of conflicts already, so that's kind of part of the job. I don't think it makes sense to anticipate all of this since there's a chance that PR won't be merged.

IMO we should merge either PR first and just solve the conflicts later. I also think it makes sense to apply changes to deprecated methods since they'll remain a part of the codebase.

Reply:
PS: We are essentially wasting time on conflicts already, so we're not gaining anything.

Reply:
@uuf6429 to be fair, I'd think it was unreasonable if I'd asked you to fix the conflicts. Picking them out and proposing them as changes (and offering to commit them) is not that different to waiting and one of us resolving the conflicts at merge time. And in this specific suggestion, it's not just that it's a conflict but that the PR is adding a redundant blank line (left over from previous additions/reverts).

The other PR will definitely be merged, there were only a couple of small details left to tackle when I reviewed this PR this weekend, and now only one small point blocking it.

Up to you what you think is most efficient, I have no objection to committing the reverts & merging this, or you can wait and rebase once #350 is merged.

Reply:
The blank line was intentional :D Usually I'm putting one between each set of types of symbols in a class (uses, constants, properties, methods) - with methods and sometimes properties having a blank line for each symbol.

Reply:
I'll wait until the other is merged, because then I can be sure these new changes are applied correctly throughout.